components/mixins/profile/profile-personal-finance-details.schema.json

```diff
@@ -53,48 +82,15 @@
               ],
               "description": "Tracks the employment status of the user."
             },
-            "xdm:filingJointly": {
-              "title": "Filing Jointly",
-              "type": "boolean",
-              "description": "Refers to a filing status for married couples that have wed before the end of the tax year."
-            },
-            "xdm:filingSeparately": {
-              "title": "Filing Separately",
-              "type": "boolean",
-              "description": "Refers to married couples who choose to record their respective incomes, exemptions, and deductions on separate tax returns."
-            },
-            "xdm:financialAccount": {
-              "title": "financeAccountId",
-              "type": "array",
-              "items": {
-                "$ref": "https://ns.adobe.com/xdm/datatypes/financial-account"
-              },
-              "description": "Financial account IDs assigned to the user, and array of Financial Account Details data type."
+            "xdm:ID": {
+              "title": "ID",
+              "type": "string",
+              "description": "Financial account ID assigned to the user."
```

Comment:
For taxid, should we include it incase it's relevant? For example if someone like an immigrant doesn't have a SSN?